app/main/views/domains.py

```diff
@@ -0,0 +1,14 @@
+from flask import jsonify, request
+
+from app.api.helpers import abort
+from app.main import main
+from app.models import Domain
+
+
+@main.route('/domain/prices/maximum', methods=['GET'])
```

Comment:
would it be easier to do a route `/domains` that just returns data for all data from domain table?